src/Bicep.Core.UnitTests/Modules/OciArtifactModuleReferenceTests.cs

```diff
@@ -51,7 +51,7 @@ public void ValidReferencesShouldParseCorrectly(ValidCase @case)
                 parsed.Repository.Should().Be(@case.ExpectedRepository);
                 parsed.Tag.Should().Be(@case.ExpectedTag);
                 parsed.Digest.Should().Be(@case.ExpectedDigest);
-                parsed.ArtifactId.Should().Be(@case.Value);
+                parsed.UnqualifiedReference.Should().Be(@case.Value);
```

Comment:
What's the intended difference between an artifact ID and an unqualified reference?

Reply:
The specific implementation of `OciModuleReference` exposes a member `ArtifactId`, however these objects are handled by callers as `ModuleReference` which exposes the `UnqualifiedReference` member which is the address to the module with the scheme stripped. `OciModuleReference.UnqualifiedReference => ArtifactId`, however for other children of `ModuleReference` `UnqualifiedReference` may be something else.